packages/common/src/images/AlertCircle.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+
+import styled from 'styled-components'
+import infoCircle from '@frontend/common/src/images/alert-image.svg'
+
+const Logo = styled.img`
+  height: 15px;
+  width: 15px;
+`
+
+const AlertCircle: React.FC = () => {
+  return <Logo src={infoCircle} alt="Logo Uppmärksamma" />
```

Comment:
I've followed the requirements from the UX-framework which says: Alternativ text (ALT-text) när ikonen saknar kompletterande text. Shouldn't this apply for this logo?

Reply:
Well, it depends on the context. Purely decorative images shouldn't have alt text. I tend to use the heuristic "does it add any valuable information" to determine whether or not to include it. Remember that the information is going to be presented to users of non-visual browsers, e.g. screen readers are going to read it aloud - will it make sense in that case?
This resource can help with the determination: https://www.digg.se/om-oss/nyheter/2021/textalternativ-for-bilder--information-nar-inte-bilden-kan-visas

Reply:
Why is it purley decorative since the purpose of it being there is to make the user aware of something?

Reply:
As I said, it depends upon the context. These kinds of icons (alert/info) are usually a visual aid and not necessarily required in a non-visual representation. For the purposes of webcert it might not matter, I don't know what portion of webcert's users are so visually (or cognitively) impaired that they need to make use of alternative content.